functions/uploadFile.ts

```diff
@@ -0,0 +1,71 @@
+import AWS, { S3, AWSError } from 'aws-sdk';
+import '@twilio-labs/serverless-runtime-types';
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import { responseWithCors, bindResolve, error500, success } from '@tech-matters/serverless-helpers';
+
+const TokenValidator = require('twilio-flex-token-validator').functionValidator;
+
+export type Body = {
+  fileBase64: string;
+  fileName: string;
+};
+
+type EnvVars = {
+  ASELO_APP_ACESS_KEY: string;
+  ASELO_APP_SECRET_KEY: string;
+  S3_BUCKET: string;
+};
+
+const sendObject = async (s3Client: S3, uploadParams: any) => {
+  return new Promise<AWS.S3.PutObjectOutput>((resolve, reject) => {
+    s3Client.putObject(uploadParams, (err: AWSError, data: S3.Types.PutObjectOutput) => {
+      if (err) {
+        reject(err);
+      }
+
+      resolve(data);
+    });
+  });
+};
+
+export const handler: ServerlessFunctionSignature = TokenValidator(
+  async (context: Context<EnvVars>, event: Body, callback: ServerlessCallback) => {
+    const response = responseWithCors();
+    const resolve = bindResolve(callback)(response);
+
+    try {
+      const { fileBase64, fileName } = event;
```

Comment:
So this entire function, instead of receiving the document's binary and uploading, will just fetch the presigned URL and send it back to user? I'm gessing based on [this documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/PresignedUrlUploadObject.html).